# 変換する対象のトピック名
map_topic_name: /map

# クラウドデータの配信トピック名
cloud_topic_name: /map_cloud

# 専有格子地図のしきい値となる専有確率
occupied_thresh:  50.0

# フレーム名
global_frame: lidar

# Lidarセンサーのスキャン位置の高さ
sensor_height:  0.35

# 配信回数
publish_times: 1